src/battery-limit.tsx

```
@@ -0,0 +1,49 @@
+import { showHUD, Form, ActionPanel, Action, getPreferenceValues, useNavigation } from "@raycast/api";
```

ESLint

[error] 'getPreferenceValues' is defined but never used.

(@typescript-eslint/no-unused-vars)